addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -403,27 +407,37 @@
     </t>
 </template>
 
-<!-- Edit Options -->
+<!-- Button to configure Tickets -->
+<template id="registration_configure_tickets_button" name="Registration Configure Ticket Button">
+    <div class="o_not_editable ml-2" role="link">
+        <a t-attf-href="/web#id=#{event.id}&amp;view_type=form&amp;model=event.event">
+            <i class="fa fa-gear mr-1" role="img" aria-label="Configure" title="Configure event tickets"/>
+            <em>Configure Tickets</em>
+        </a>
+    </div>
+</template>
+
+<!-- This template is called whenever the "Fold Ticket Details" option is checked on the website (Customize > Fold Tickets Details) -->
 <template id="fold_register_details" inherit_id="website_event.registration_template" active="False" customize_show="True" name="Fold ticket details">
     <xpath expr="//div[@id='o_wevent_tickets']" position="attributes">
         <attribute name="data-folded-by-default">1</attribute>
     </xpath>
     <xpath expr="//div[@id='registered']" position="attributes">
         <attribute name="class">ml-auto pr-3</attribute>
     </xpath>
-    <xpath expr="//div[@id='price-range']" position="attributes">
-        <attribute name="class">ml-0 pr-3</attribute>
-    </xpath>
     <xpath expr="//div[@id='o_wevent_tickets_collapse']" position="attributes">
         <attribute name="class">collapse</attribute>
     </xpath>
+    <xpath expr="//span[@id='tickets-title']" position="attributes">
+        <attribute name="class">py-2 o_wevent_registration_title text-left d-none</attribute>
+    </xpath>
+    <xpath expr="//div[@id='price-range']" position="attributes">
+        <attribute name="class">o_price_range</attribute>
+    </xpath>
```

Comment:
I think you can safely add this class directly into the DOM item to simplify.
Even if it's actually only used when the option is ticked.

And you can rename to `o_wevent_price_range` to keep the same prefix as others.

Reply:
I'll rename. However, I'm not only adding this class in this case, I'm also replacing the "d-none". For the current logic, I can't just get rid of this xpath, so I figured I might as well just write the right classes where it belongs

Reply:
Mmh then I would rather have "o_wevent_price_range d-none" on the base DOM element and make an override based on the class.
The "o_wevent_price_range" is used to target this element so I think the code is more clear if it's always there (even if only used when the folding option is enabled).